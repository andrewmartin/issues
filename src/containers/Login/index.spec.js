import React from 'react';
import renderer from 'react-test-renderer';
import { Login } from './index';

const user = {
  serverError: null,
};

it('renders Login', () => {
  const tree = renderer.create(<Login user={user} />).toJSON();
  expect(tree).toMatchSnapshot();
});
