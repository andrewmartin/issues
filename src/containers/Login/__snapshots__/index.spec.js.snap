// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Login 1`] = `
<main
  className="layout"
>
  <div
    className="container"
  >
    <h2>
      Login
    </h2>
    <form
      className="form"
      onSubmit={[Function]}
    >
      <input
        className="input input"
        onChange={[Function]}
        placeholder="Your Github API Key"
        required={true}
        type="text"
        value=""
      />
      <button
        className="button"
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
</main>
`;
